### DISCUSSION TITLE
Use of index.html

### DISCUSSION THREAD

**Discussion by AsuraKev** [web-infra-dev/modern.js#4674] [Q&A]
Hi there,

Is there an index.html in the project template? I have a use case where I ll need to explicitly put some html scripts e.g google api script directly on the html header. Any idea if this is achievable?

thanks

**Comment by lucas-0liveira** [comment]
hello, sim. it's possible to add an index.html file to the project template. This will allow you to include HTML scripts directly in the document's header.

To add the index.html file to your project, follow these steps:

Open your code editor or development environment.

Create a new file named index.html in the folder of your project or in the location where you are keeping your HTML files.

Now, you can include the HTML scripts you need in the header of the index.html file.

I hope this helps
